### PR TITLE
T21701

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T21701

### CODE REVIEW

**Comment by xjuan** [comment]
screenshot from 2018-03-27 15-45-56
Gallery looks fine, but news looks broken
should not the background on news also be transparent?

**Comment by tchx84** [comment]
@xjuan yeah, I think it's missing that in the news overries... I will fix it.

File: data/css/modules/window/_simple.scss [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
     scrollbar {
         padding: 3px;
-        background-color: transparentize(black, 1 - 0.20);
+        background-color: #d6d7d8;
```
Comment (tchx84) [inline data/css/modules/window/_simple.scss]: I think this needs more work, since it doesn't look too good on other presets/themes:

screenshot from 2018-03-27 15-27-59
screenshot from 2018-03-27 15-27-38

I am going to try to revert these changes and try to extract the core improvements without affecting other presets/themes.

File: data/css/news.scss [hunk 2]
```diff
@@ -118,3 +118,8 @@ $support-font: 'Fira Sans' !default;
         }
     }
 }
+
+.WindowSimple scrollbar {
+    // margin-top is for being covered by Layout.TopMenu
+    margin-top: 70px;
```
Comment (tchx84) [inline data/css/news.scss]: I think this wasn't tested against the default news, I will tweak it as part of the rework.

screenshot from 2018-03-27 15-32-35

File: data/css/news.scss [hunk 3]
```diff
@@ -118,3 +118,10 @@ $support-font: 'Fira Sans' !default;
         }
     }
 }
+
+.WindowSimple {
+    scrollbar {
+        // margin-top is for being covered by Layout.TopMenu
+        margin-top: 50px;
```
Comment (xjuan) [inline data/css/news.scss]: We need to open a new task to handle this properly.
Ideally this should be the responsibility of the container that packs the topbar over the webview

**Comment by xjuan** [comment]
CR pass

File: data/css/news.scss (hunk 3 above)
Comment (ptomato) [reply data/css/news.scss]: If I understand correctly what you mean, then I agree! Can you make sure to open this issue?
